PraceDomowe/Ly_HoangThien_ Homework no.5/Homework no.5 _ Final Version.ipynb

```diff
@@ -0,0 +1,1886 @@
+{
```

Comment:
<p>I used the <code>Id</code> column, not <code>AccountId</code> but I understand that it's ambiguous. -0.2 points for the fact that the solution is not automatic (you had to manually input value 101 in the cell above) and -0.1 for the fact that the result is a <code>pd.DataFrame</code>, not a <code>string</code>.</p><p> </p><p><code>0.7 points</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>